editors/code/src/Commands/syntaxTree.ts

```diff
@@ -41,16 +41,16 @@ export function syntaxTreeCommand(ctx: CtxInit): Cmd {
     ): Promise<string> {
       const params: SyntaxTreeParams = {};
       const doc = vscode.window.activeTextEditor?.document;
-      if (doc === undefined) {
+      if (!doc) {
```

Comment:
Please revert this. Using a negative expression on types can lead to weird JS behaviours which have cause problems in the past.

Reply:
Got it!